ideas/311-desktop-team-proposal.md

```diff
@@ -0,0 +1,124 @@
+  * [use peer counts to detect online\offline state](https://github.com/status-im/status-react/issues/6961)
+  * [No new messages after going back from sleep](https://github.com/status-im/status-react/issues/6396)
+  * [Security and privacy review](Desktop security and privacy review)
+  * [Context menu in input fields](https://github.com/status-im/status-react/issues/6571)
+  * [Add design for menus](https://github.com/status-im/status-react/issues/4434)
+  * [Fix scroll stuttering](https://github.com/status-im/status-react/issues/6570)
+  * [Add dev mode and network switching](https://github.com/status-im/status-react/issues/6477)
+  * [UI fixes for bubble-less chat area](https://github.com/status-im/status-react/issues/6506)
+  * [Fix group chats creation](https://github.com/status-im/status-react/issues/6607)
+  * [Confirmation dialog on logout](https://github.com/status-im/status-react/issues/4977)
+  * [Custom url support on linux](https://github.com/status-im/status-react/issues/6394)
+  * [Add custom mailserver](https://github.com/status-im/status-react/issues/6110)
+  * [Edit profile picture](https://github.com/status-im/status-react/issues/5456)
+  * [Remove code duplication in Jenkinsfiles](https://github.com/status-im/status-react/issues/5424)
+  * [Persist scroll position after navigation through the app](https://github.com/status-im/status-react/issues/5852)
+  * [Add jenkins job for e2e tests](https://github.com/status-im/status-react/issues/5859)
+  * [Apply onboarding designs](https://github.com/status-im/status-react/issues/4418)
+ * **what existing desktop issues affect its usability most (in functionality or UI)**?  
+ * **how many users outside of Status we have? (optional)**
+ * **how can we get user feedback about desktop**
+ * **what features mostly anticipated by users?**
+ * **what ui/ux input needed**
+ * **what should be done in `react-native-desktop` to support status desktop tasks**
+  * [Get rid of ubuntu-server and embed js engine in react-native-desktop](https://github.com/status-im/status-react/issues/6175)
+  * [File open dialog](https://github.com/status-im/react-native-desktop/issues/377)
+  * [Fix remote js debugging](https://github.com/status-im/react-native-desktop/issues/392)
+  * Improve UI responsiveness
+ * **what should be done in `react-native-desktop` to make its maintenance easier**
+  * port to react-native 0.57. That will give us a possibility to make it an [out-of-tree](https://facebook.github.io/react-native/docs/out-of-tree-platforms) platform and facilitate future upgrades.
+    * PR with updated version [already in react-native-desktop](https://github.com/status-im/react-native-desktop/pull/422)
+    * [PR for testing it with status-react](https://github.com/status-im/status-react/pull/6983)
+    * PR for mobile [in progress](https://github.com/status-im/status-react/pull/6951)
+  * More documentation/tutorials for external contributors
+
+
+`Research output:`
```

Comment:
as well as bounties and user stories optimized for

Reply:
added